app/controllers/identify_controller.rb

```diff
@@ -101,7 +102,8 @@ def allowed_referrer?(uri)
 
   def log_certificate(cert)
     validation_result = cert.validate_cert(is_leaf: true)
-
+    login_certs_openssl_result = openssl_validate(cert.to_pem, Rails.root.join(IdentityConfig.store.login_certificate_bundle_file).to_s)
+    ficam_certs_openssl_result = openssl_validate(cert.to_pem, Rails.root.join(IdentityConfig.store.login_certificate_bundle_file).to_s)
```

Comment:
what's the difference between these two lines? looks like they call the same methods with the same arguments

Reply:
🤦🏼‍♂️ 

Supposed to be using the two different cert bundles to be able to compare, fixed in 1b483aa